modules/gateway/gateway_test.go

```diff
@@ -0,0 +1,76 @@
+package gateway
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/consensus"
+	"github.com/NebulousLabs/Sia/modules"
+)
+
+var port = 9001
```

Comment:
close() might not be good enough anyway. iirc the test framework will run different package tests in parallel.